templates/dashboard.yaml

```diff
@@ -79,16 +13,10 @@ metadata:
 spec:
   type: {{ .Values.serviceType | quote }}
   selector:
-    coder.deployment: dashboard
+    coder.deployment: {{ include "coder.serviceName" . }}
```

Comment:
Wouldn't this still be the dashboard?

Reply:
This changes which pods the service points to. We want it to point to the coderd pods since that's where the dashboard is